### DISCUSSION TITLE
How to load custom font family?

### DISCUSSION THREAD

**Discussion by prakharbhardwaj** [twilio-labs/paste#3931] [Q&A] [answered]
Hello team,

I'm looking for a cleaner solution to load a custom font i.e. by loading font files (e.g. TTInterphasesPro-Regular.ttf) to the chat. 

I tried to look at the [documentation](https://paste.twilio.design/customization/creating-a-custom-theme), but couldn't find a relevant answer.

FYI: I want to use it in new Twilio Webchat App.

Thank you.

**Comment by serifluous** [comment, marked as answer]
Hey @prakharbhardwaj ! We recommend reaching out to your Twilio Flex Account Manager for support on your instance of the webchat app.

There's a section at the bottom of our theme customization docs about changing the font in your custom theme with a CodeSandbox example that might help: https://paste.twilio.design/customization/creating-a-custom-theme#customizating-typography

However the way you load your font file into your app will depend on your specific app and font provider, so your account manager may be able to help more than we can!
